DigitalOcean.API/Helpers/JsonDeserializationHelper.cs

```diff
@@ -0,0 +1,15 @@
+using Newtonsoft.Json.Linq;
+using RestSharp;
+
+namespace DigitalOcean.API.Helpers {
+    internal class JsonDeserializationHelper {
+
+        public static T DeserializeWithRootElementName<T>(string json, string rootElement) {
```

Comment:
I think we might want to slightly change this extension though to instead take a `JObject` rather than the raw json `String`, so we don't have to desrialize the same string multiple times. And just use the JObject to extra necessary root element, then calling `ToObject()` with our JsonSerializer.